ci-operator/step-registry/ipi/conf/tang/ipi-conf-tang-commands.sh

```diff
@@ -57,10 +57,11 @@ boot_device:
         thumbprint: ${TANG_SERVER_KEY}
     threshold: 1
 EOF
-    /tmp/butane "${workdir}/${role}_tang_disk_encryption.bu" > "${workdir}/manifest_${role}_tang_disk_encryption.yml" || ret_code=$?
+    /tmp/butane "${workdir}/${role}_tang_disk_encryption.bu" > "${workdir}/manifest_${role}_tang_disk_encryption.yml"
+    ret_code=$?
```

Comment:
need to merge line 61 into line 60 like before, else script will exit when `butane` run failed.

Reply:
Yeah, you are right, I disabled `errexit` for now.